examples/customDataHandlers.js

```diff
@@ -85,13 +85,51 @@ let customDataHandlers = {
     return series;
   },
 
+
+
+  // @TODO use data with dashboardData.
   getTableData: function (componentData, dashboardData, handler, e, appliedFilters, pipelineData) {
-    console.log(arguments);
-    if (dashboardData.climateData) {
-      return dashboardData.climateData;
+
+    // Gey data.
+    const data = dashboardData.climateData || [];
```

Comment:
"climateData" seems a bit odd here... is this a specific implementation?

Reply:
it's also not a parameter

Reply:
It's the data that has been used for all of the demos, so it's common to the example page.